draft-trammell-taps-post-sockets.md

```diff
@@ -164,8 +164,8 @@ Post replaces the traditional SOCK_STREAM abstraction with an Message
 abstraction, which can be seen as a generalization of the Stream Control
 Transmission Protocol's {{RFC4960}} SOCK_SEQPACKET service. Messages are sent
 and received on Carriers, which logically group Messages for transmission and
-reception. For backward compatibility, these Carriers can also be opened as
-Streams, presenting a file-like interface to the network as with SOCK_STREAM.
+reception. For backward compatibility, byte stream protocols are represented
+as a Carrier with a single long message in each direction.
```

Comment:
Maybe add a comment that says the end is determined by the end of the connection (FIN)?

Reply:
+1 to something combining colin's and mirja's suggestions here...